packages/bpk-component-badge/src/BpkBadge.js

```diff
@@ -42,7 +44,14 @@ const badgeTypeClassNames = {
   [BADGE_TYPES.outline]: getClassName('bpk-badge--outline'),
 };
 
-const BpkBadge = props => {
+export type Props = {
+  type: ?$Keys<typeof BADGE_TYPES>,
```

Comment:
One last thing - sorry, I should have mentioned this earlier - this also shouldn't be optional, because there must always be a `type`.

It's a bit confusing and it took me a bit to get my head around, but think of it like this: if it can be `null`, it's ok to be optional. As `type` has a non-null `defaultProps` declaration, it can never be null.

Reply:
Cool. Have also now done `BadgeLayout.js`

As external consumers don't use the `BadgeLayout`, should I remove the old `prop-types`

Reply:
It's up to you really. Personally I would leave it in, but you're right that it's not technically required now!